amlip_cpp/test/blackbox/communication/collaborative_learning/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
NIT: be careful with this. The copyright of each file should be the year of creation